aten/src/ATen/native/Math.h

```diff
@@ -384,7 +377,7 @@ static inline float calc_digamma(float x) {
   float y = 0;
   if (x < 1.0e17f) {
     float z = 1 / (x * x);
-    y = z * polevlf(z, A, 6);
+    y = z * polevl(z, A, 6);
```

Comment:
What does this change do?

Reply:
Previously this file had `double polevl(double x, double *A, size_t len)` for double and `float polevlf(float x, float *A, size_t len)` for floating inputs. We have no replaced them with a template `polevl`. Hence the change.